website/docs/quickstart.md

```diff
@@ -117,17 +122,31 @@ The Subo REPL includes a proxy that makes it easy to connect the hosted editor t
 
 The `local.suborbital.network` subdomain points to `127.0.0.1`, i.e. `localhost`. You may need to substitute a different hostname or IP address depending on your particular network setup.
 
+## Create a tenant (user)
+
+Suborbital lets an application's users create their own secure, sandboxed plugins, carefully isolated from the core of the system and one another. For this reason, we will create a new tenant, which is a user account with its own plugins inside Suborbital. Our application will then connect the tenant with one of its own internally maintained users.
```

Comment:
I find this confusing. Not your fault -- I find the plugin hierarchy itself confusing -- but maybe there's a way to rephrase this?

I especially tripped on "okay, a tenant is a user... but wait, the tenant has users, too? And they're internally maintained?"

Reply:
I'd be good with just deleting the second sentence and letting the first stand alone, maybe linked to the glossary? The rest of the info is more advanced than we strictly *need* to include in a quickstart.

Reply:
In fact we don't need this section at all, since creating a tenant happens at the moment of editor token creation instead.